modules/nw-ovn-k-adminnetwork-policy.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn-k-network-policy.adoc
+
+:_mod-docs-content-type: <Concept>
+[id="AdminNetworkPolicy"_{context}"]
+= AdminNetworkPolicy
+
+`AdminNetworkPolicy` (ANP) is a cluster-scoped custom resource definitions (CRDs). As an {product-title} administrator, you can use `AdminNetworkPolicy` to secure your cluster in the following ways:
+
+* You can create cluster-scoped policies prior to namespaces being created.
```

Comment:
This feels off to me, how about just say: You can create ANPs even before namespaces are created thus enforce security rules earlier on or something....
First point and second  point have no correlation IMHO..